bin/kubeconfig.bash

```diff
@@ -107,7 +107,7 @@ dev)
   log_info "Adding dev ${serviceAccount} context to wc-config"
 
   token=$(with_kubeconfig "${kubeconfig}" kubectl get secrets secret-"${serviceAccount}" -ojsonpath="{.data.token}" | base64 -d)
-  cluster_name=$(yq '.global.clusterName' "${cluster_config}")
+  cluster_name=$(kubectl config view -o jsonpath='{.contexts[?(@.name == "'"$(kubectl config current-context)"'")].context.cluster}')
```

Comment:
Which kubeconfig are these `kubectl` calls using? I'm not seeing any `KUBECONFIG` defined so I'm assuming the one in `~/.kube/config`, is that intentional, given how it used the merged wc-config before?

Reply:
> Which kubeconfig are these `kubectl` calls using? I'm not seeing any `KUBECONFIG` defined so I'm assuming the one in `~/.kube/config`, is that intentional, given how it used the merged wc-config before?

Nope you're right, the other commands used --kubeconfig to set it. Fixed now: https://github.com/elastisys/compliantkubernetes-apps/pull/2514/commits/1c627373b7f4aad4e0c4ce0bc031294b49c59815